content/applications/hr/referrals/alerts.rst

```diff
@@ -0,0 +1,50 @@
+======
+Alerts
+======
+
+In the *Referrals* application, it is possible to post a message, also referred to as an *alert*, at
+the top of the dashboard to share important information with users. These alerts appear as a thin
+semi-transparent banner, with the word :guilabel:`New!` appearing on the far left. The text for the
+alert is in the center of the banner, and on the far right side is an :guilabel:`X`.
```

Comment:
```suggestion
semi-transparent banner, with the word :guilabel:`New!` appearing on the far left. The text for the
alert is in the center of the banner, and on the far right side is an :guilabel:`X`.
```
I would remove the last sentence. I think it's unnecessary to say where the text appears on the banner. The 'X' on the right side can be covered later in the documentation by an instruction telling the user how to dismiss the banner.